backend/src/db/migrations/20240205200732_invite-only.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  const isTablePresent = await knex.schema.hasTable(TableName.SuperAdmin);
+  if (isTablePresent) {
+    await knex.schema.alterTable(TableName.SuperAdmin, (t) => {
+      t.boolean("inviteOnlySignUp").defaultTo(false);
+      t.string("allowSpecificDomainSignUp");
```

Comment:
Different departments or branches of a large organization might use distinct domains so not all will be the same. Can you add option for multiple?

Reply:
- Do want it as a comma separated or Array of string in the database?
- Same input element or Input tags?

Reply:
Added it as a comma separated for now.